### ISSUE TITLE
Can I run kafka-manager without zk config?

### ISSUE THREAD

**Issue by xikunyang** [yahoo/kafka-manager#500]
For some reason, I just have broker server address.

**Comment by montana-ua** [comment]
No. You must specify zookeeper connection endpoint because KM store configuration in ZK DB.

**Comment by walkafwalka** [comment]
Doesn't this seem like a bit of an overkill? A configuration file should be more than enough.
